.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: generic
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - docker pull fitiotlab/iot-lab-gateway:latest
+  - docker build -t iot-lab-gateway .
+  - docker build -t iot-lab-gateway-test tests
+
+script:
+  - >
+    docker run -v gateway_code:/home/iot-lab-gateway/gateway_code
```

Comment:
I tried the commands locally and this one fails. It looks like my changes are not taken into account. But maybe I missed something

Reply:
what's the error ?
you've built both images before the docker run ?
```
docker build -t iot-lab-gateway-test tests
docker build -t iot-lab-gateway .
```

Reply:
yes, I have errors (always the same, even if I change the setup.cfg) related to coverage:
```
nose.plugins.cover: WARNING: Failed to generate HTML report: No source for code: '/home/iot-lab-gateway/gateway_code/control_node/__init__.py'.
Aborting report output, consider using -i.
Traceback (most recent call last):
  File "setup.py", line 199, in <module>
    install_requires=INSTALL_REQUIRES)
  File "/home/iot-lab-gateway/.tox/check/local/lib/python2.7/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup

... (long traceback)

    raise NoSource(exc_msg)
coverage.misc.NoSource: No source for code: '/home/iot-lab-gateway/gateway_code/control_node/__init__.py'.
Aborting report output, consider using -i.
ERROR: InvocationError: '/home/iot-lab-gateway/.tox/check/bin/python setup.py nosetests'
___________________________________ summary ____________________________________
  copying: commands succeeded
ERROR:   check: commands failed
```

Reply:
I'm running: docker run -v gateway_code:/home/iot-lab-gateway/gateway_code -v $CWD/tests_utils/cfg_dir:/var/local/config -e BOARD_TYPE=m3 -e CONTROL_NODE_TYPE=no -e HOSTNAME=m3-00 -e IOTLAB_GATEWAY_NO_INTEGRATION_TESTS --privileged iot-lab-gateway-test

Reply:
Fixed, the problem was on my side: I had to rebuild the containers. And this is a problem for me (non blocking, but will need to be fixed). Because this means each time I change code locally, I have to rebuild everything: is my local the gateway_code mapped in the container ? I thought it was.

Reply:
setup.cfg is not passed to the container, with `-v gateway_code:/home/iot-lab-gateway/gateway_code`  you mount only gateway_code , try `-v $CWD:/home/iot-lab-gateway`

Reply:
Ah yes !

Reply:
OK, I'm sure this would be more intuitive if we mounted the whole iot-lab-gateway folder instead of only gateway_code. Not sure about the implications